PhysicsTools/PythonAnalysis/test/imports.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+set -e
+
+echo ">>> Create temporary directory for cache"
+TEST_TMPDIR=$(mktemp -d /tmp/cmssw_theano.XXXXXXX)
+
+echo ">>> Change default behaviour for Theano"
+export THEANO_FLAGS="device=cpu,force_device=True,base_compiledir=$TEST_TMPDIR"
+
+echo ">>> Theano configuration for testing:"
+python -c 'import theano; print(theano.config)'
+
+echo ">>> Cleaning compile cache"
+theano-cache clear
+
```

Comment:
@davidlt, why not move this above code in a single file and call run that from all these tests?

Reply:
We could, but didn't see a big need so far (small enough change). Who knows, maybe original author will want to configure each test in a different way.

But we can do it.